options/locale/locale_en-US.ini

```diff
@@ -1391,18 +1391,15 @@ projects.type.basic_kanban = "Basic Kanban"
 projects.type.bug_triage = "Bug Triage"
 projects.template.desc = "Template"
 projects.template.desc_helper = "Select a project template to get started"
-projects.type.uncategorized = Uncategorized
 projects.column.edit = "Edit Column"
 projects.column.edit_title = "Name"
 projects.column.new_title = "Name"
 projects.column.new_submit = "Create Column"
 projects.column.new = "New Column"
 projects.column.set_default = "Set Default"
 projects.column.set_default_desc = "Set this column as default for uncategorized issues and pulls"
-projects.column.unset_default = "Unset Default"
-projects.column.unset_default_desc = "Unset this column as default"
 projects.column.delete = "Delete Column"
-projects.column.deletion_desc = "Deleting a project column moves all related issues to 'Uncategorized'. Continue?"
+projects.column.deletion_desc = "Deleting a project column moves all related issues to the default column. Continue?"
```

Comment:
It needs a new key

Reply:
Why? New translations for existing keys now invalidate the old ones, so it isn't necessary.

Reply:
NO. I have explained many times, it doesn't "invalidate", unless I remember wrong or Crowdin has changed behaviors.

Especially in your issue #27700, I have said clearly about this problem. https://github.com/go-gitea/gitea/issues/27700#issuecomment-1771892952

Let's see a real example: `pulls.push_rejected` has been changed from "Merge failed" to "Push failed" in #29537, but in other languages, they are still the old translation, eg: `pulls.push_rejected=合并失败 (merge failed)`

I won't strongly say objection for this, because I guess I don't want to be considered to be unfriendly during code review, and non-English translations don't bother me. I am suggesting just because my philosophy is "do the right thing".

Reply:
Crowdin sync seems broken completely currently - deleted translation keys stay in place in the translations currently, and it needs to be fixed ASAP. But invalidation works, I've seen it multiple times

Reply:
How it works? Why `pulls.push_rejected` hasn't been updated in weeks .....

Reply:
Currently, nothing works because (at least it seems to me) like our translations aren't synced to Crowdin at all. @techknowlogick should know more... we should switch to the Crowdin GitHub action ASAP

Reply:
Hmm, it seems that we have 2 problems now. 

1. The crowdin sync stops working
2. The translations don't get invalidated

I know the "don't get invalidated" because when I was writing the "backport-locales.go" (#23633), I found that many translations are outdated, for example: `%s` numbers don't match. It is a quite longstanding problem (and I have mentioned it many times 😢 )

Reply:
There was a long time when the translations were not invalidated, but then techknowlogick changed something on the sync side and newly changed translations were invalidated, but not the old ones. That's another issue I would like to solve soon as part of #27700

Reply:
Well, I didn't know the "something" has been changed, nor whether it really works as expectation. And actually I would expect to "git blame" first to fix the old outdated keys, before correct the keys invalidation ......

Reply:
It will happen soon, but outside of this PR obviously :)

Reply:
Hmm .... one more question: if there is no fix before 1.22, then non-English users would see incorrect sentences?

Reply:
We need a fix before 1.22, otherwise there's A LOT of wrong and/or missing translation keys....